tests/Functional/NginxTest.php

```diff
@@ -74,7 +74,10 @@ public function testRefresh()
         $nginx->refresh('/cache.php')->flush();
         usleep(1000);
         $refreshed = $this->getResponse('/cache.php');
-        $this->assertGreaterThan((float) $response->getBody(true), (float) $refreshed->getBody(true));
+        $this->assertGreaterThan(
+            (float)(string) $response->getBody(),
```

Comment:
We need `(string)` to get the response body from the [StreamInterface](https://github.com/php-fig/http-message/blob/master/src/StreamInterface.php#L28), then the original `(float)` for correct timestamp comparison.